### ISSUE TITLE
Jeremy to poke USGS about stream height dat (lots of these)

### ISSUE THREAD

**Issue by rweigel** [hapi-server/hapi-tasks#6]

**Comment by jvandegriff** [comment]
Jeremy to follow up this week with more email

**Comment by rweigel** [comment]
This paper summarizes USGS water-related data: https://digitalcommons.usu.edu/cgi/viewcontent.cgi?article=4823&context=cee_facpub

This is an interesting case of time series data organized differently, but it does probably map to HAPI. There are millions of instruments ("With millions of groundwater levels collected at sites across the nation, the amount of data available is very large"). As a result, the catalog will contain millions of unique site/instrument pairs. In addition, each site does not have the same instruments, and the instruments change. For example, https://waterservices.usgs.gov/nwis/site/?format=rdb&sites=01646500&seriesCatalogOutput=true

**Comment by jbfaden** [comment]
Yes, should be an interesting demonstration server.  I've gotten the go-ahead from USGS people, I just need to find time to start playing with this.

**Comment by jvandegriff** [comment]
Consider looking up people at the AGU to ask about this.

Find folks who are using the data and see how they might use or benefit from a HAPI server.
